### PR TITLE
Add project presentations 2023

### PR DESCRIPTION
**Pull request by zanemit**
This adds a directory `projects/past-projects` that contains the project presentations of all groups that have consented to public sharing of their projects and a jupyter notebook per project topic to render the submitted slides.

`projects/docs/datasets_overview.md` was also edited because the paths were not working and the albedo project name was not consistent with other instances.

Updated `project_materials.yml` with the desired page structure in build.

### CODE REVIEW

File: projects/docs/datasets_overview.md [hunk 1]
```diff
@@ -5,12 +5,12 @@ You have been matched with 4-6 students from your pod to form a project group. B
...
 You can find project materials in the left side panel or by clicking on the project titles below.
-- [Sea level rise](./project-notebooks/Sea_level_rise.ipynb)
-- [Regional precipitation variability and extreme events](./project-notebooks/Regional_precipitation_variability.ipynb)
-- [Change in Earth’s albedo and its dependence on land cover changes in the past 20 years](./project-notebooks/Surface_albedo_and_land_use.ipynb)
-- [Monitoring and mapping wildfires using satellite data](./project-notebooks/Wildfires_and_burnt_areas.ipynb)
-- [Ocean acidification](./project-notebooks/Ocean_acidification.ipynb)
-- [The impact of ENSO on precipitation and temperature](./project-notebooks/ENSO_impact_on_precipitation_and_temperature.ipynb)
-- [Heatwaves: Assessing the dynamic interactions of the atmosphere and land](./project-notebooks/Heatwaves.ipynb)
+- [Sea level rise](../project-notebooks/Sea_level_rise.ipynb)
+- [Regional precipitation variability and extreme events](../project-notebooks/Regional_precipitation_variability.ipynb)
```
Comment (WesleyTheGeolien) [inline projects/docs/datasets_overview.md:9]: @zanemit I think this needs to be from root of the projects so it should be `projects/project-notebooks/...` could you update the others? Is there a reason you believe it should be `../` those were my thoughts of the top of my head


```suggestion
- [Regional precipitation variability and extreme events](projects/project-notebooks/Regional_precipitation_variability.ipynb)
```

File: projects/docs/past_projects_overview.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Past projects
+
+We are proud to showcase the project presentations our students prepared in previous iterations of *Computational Tools for Climate Science*. 
+
+You can browse their presentation slides by project topic:
+- [Sea level rise](../past-projects/Sea_level_rise.ipynb)
```
Comment (WesleyTheGeolien) [inline projects/docs/past_projects_overview.md:6]: same here for `projects/past-projects` I would guess

File: projects/past-projects/project-presentations/2023ALB_Changes_in_land_cover_Albedo_and_temperature_in_Mato_Grosso.pdf
Comment (WesleyTheGeolien) [inline projects/past-projects/project-presentations/2023ALB_Changes_in_land_cover_Albedo_and_temperature_in_Mato_Grosso.pdf:1]: I'm generally unsure about commiting large files to the repo as it bloats the history it is bad practice .... not sure we have a better choice at the moment though ....

File: projects/project_materials.yml [hunk 3]
```diff
@@ -23,3 +23,20 @@
     title: "Changes in Land Cover: Albedo and Carbon Sequestration"
 - file: projects/docs/continuing_your_project_after_the_course.md
   title: Continuing your project after the course
+- file: projects/docs/past_projects_overview.md
+  title: Past projects
+  section:
```
Comment (WesleyTheGeolien) [inline projects/project_materials.yml:28]:
```suggestion
  sections:
```

**Review by WesleyTheGeolien — CHANGES_REQUESTED** [review]
Looks good @zanemit just a few comments I  left.

Don't worry about CI check that is still wrong but on my side

File: projects/docs/datasets_overview.md (hunk 1 above)
Comment (zanemit) [reply projects/docs/datasets_overview.md:9]: Hm. I haven't managed to set up a local build, but ctrl+click in VScode opens the correct file with the `..\` syntax. It cannot find the file with the syntax we had (which doesn't work in the built book) or the one you have suggested. The location of this file is `projects/docs`, so I think it makes sense that it needs to be guided one directory up. `projects/project-notebooks/<filename>` would be the syntax if `projects` was a folder inside `docs`, no?

File: projects/docs/past_projects_overview.md (hunk 2 above)
Comment (zanemit) [reply projects/docs/past_projects_overview.md:6]: Same reply as above :)

File: projects/past-projects/project-presentations/2023ALB_Changes_in_land_cover_Albedo_and_temperature_in_Mato_Grosso.pdf
Comment (zanemit) [reply projects/past-projects/project-presentations/2023ALB_Changes_in_land_cover_Albedo_and_temperature_in_Mato_Grosso.pdf:1]: Ah, sorry, I had understood that you were happy with github as a long-term solution. If you'd rather wait for the OSF expansion, which @NickHalper agreed to in principle, we can pause this for now and I can update the code to OSF/MFR + remove the presentation files from the commit once we have everything on OSF. Let me know!

File: projects/project_materials.yml (hunk 3 above)
Comment (zanemit) [reply projects/project_materials.yml]: Thanks for spotting that! Fixed it!

File: projects/docs/datasets_overview.md (hunk 1 above)
Comment (WesleyTheGeolien) [reply projects/docs/datasets_overview.md:9]: Yeah I get it works in VS code ^^ unfortunatly during the book build everything gets sym linked into the book folder and it is built from there so I do think projects is needed, see below:

image

File: projects/docs/past_projects_overview.md (hunk 2 above)
Comment (WesleyTheGeolien) [reply projects/docs/past_projects_overview.md:6]: same reply as above ^^

File: projects/docs/datasets_overview.md (hunk 1 above)
Comment (zanemit) [reply projects/docs/datasets_overview.md]: Thanks for explaining! Done now!

**Pull request merged**